source/tutorials/nixos/nixos-configuration-on-vm.md

```diff
@@ -186,14 +174,6 @@ This command builds the attribute `vm` from the `nixos-23.11` release of NixOS,
 
   Here we set `nixpkgs` to refer to a [specific version of Nixpkgs](ref-pinning-nixpkgs) and set `nix-config` to the `configuration.nix` file in the current directory.
 
-:::{admonition} NixOS
-On NixOS the `$NIX_PATH` environment variable is usually set up automatically, and there is also [a convenience command for building virtual machines](https://nixos.org/manual/nixos/stable/#sec-changing-config).
-You can use the current version of `nixpkgs` to build the virtual machine like this:
-```shell-session
-nixos-rebuild build-vm -I nixos-config=./configuration.nix
-```
```

Comment:
Why do you suggest to remove this command/section?

Reply:
`nixos-rebuild` is really intricate with how it works. When I tested it, it failed to do what one would expect because there was an `/etc/nixos/flake.nix` file..

To keep this tutorial more focused on the use case of building a VM just using Nix, we thought it made sense to avoid any potential confusion with `nixos-rebuild`.

Reply:
oh yes flakes ... ok.